ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -89,3 +89,9 @@ def shuffle(x, /):
 @from_zero_dim_arrays_to_scalar
 def standard_normal(size=None):
     return ivy.random_normal(mean=0.0, std=1.0, shape=size, dtype="float64")
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def standard_exponential(beta):
+    return ivy.gamma(1.0, beta, dtype="float64")
```

Comment:
are you sure the `standard exponential` function in Numpy behaves identically to the `gamma` function? Numpy provides both these functions separately so shouldn't there must be some difference?